### PR TITLE
Update how-to-prerequisites.md

### PR DESCRIPTION
**Pull request by akosre**
adding the requirement for http 1.1 and chunked encoding

### CODE REVIEW

**Comment by Court72** [comment]
@billmath

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by akosre** [comment]
Hello Bill,

can you approve this PR or do we need here to ask Dhanyah?

Thanks,
Akos

**Comment by billmath** [comment]
#sign-off

File: docs/identity/hybrid/cloud-sync/how-to-prerequisites.md [hunk 1]
```diff
@@ -118,6 +118,7 @@ If there's a firewall between your servers and Microsoft Entra ID, configure the
    | **8080** (optional) | Agents report their status every 10 minutes over port 8080, if port 443 is unavailable. This status is displayed in the Microsoft Entra admin center. |
 
 - If your firewall enforces rules according to the originating users, open these ports for traffic from Windows services that run as a network service.
+- Ensure that your proxy supports at least HTTP 1.1 protocol and chuncked encoding is enabled.
```
Comment (Court72) [inline docs/identity/hybrid/cloud-sync/how-to-prerequisites.md:121]:
```suggestion
- Ensure that your proxy supports at least HTTP 1.1 protocol and chunked encoding is enabled.
```

**Pull request merged**